src/ConstraintValidator/Operator/String/StringStartsWithOperatorValidator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Unleash\Client\ConstraintValidator\Operator\String;
+
+/**
+ * @internal
+ */
+final class StringStartsWithOperatorValidator extends AbstractStringOperatorValidator
+{
+    protected function validate(string $currentValue, array|string $searchInValue): bool
+    {
+        assert(is_string($searchInValue));
+
+        return str_starts_with($currentValue, $searchInValue);
```

Comment:
only from PHP 8?

Reply:
Explained in previous comments, gets downgraded to suitable expression for each version.